policies/recipes/recipedb.go

```diff
@@ -18,22 +18,60 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"encoding/json"
+	"io/ioutil"
+)
+
+var (
+	dbPath := "~/placeholder_location/swr.json"
```

Comment:
Where should the recipedb file live on the different operating systems?

Reply:
a state file that persists is probably appropriate to put somewhere under `/var/cache`, but we would want to look at the linux FHS and maybe some similar products to be sure. should be somewhere that indicates it isn't available for editing (i.e. not `/etc`) and not somewhere temporary (`/tmp`, `/var/run`, etc.)

Reply:
So I'm curious for your thoughts are but so far the best option I've found is /var/lib

Reply:
Well from talking to you and more research the best I've found is /var/lib/google/. This seems to make sense from an overall filesystem perspective and also is similar to where programs, such as apt, put similar data.